src/server/system_services/stats_aggregator.js

```diff
@@ -1157,7 +1093,7 @@ async function _perform_full_cycle() {
         })
     });
 
-    await _handle_payload(payload);
+    await _handle_payload();
```

Comment:
maybe a better name if it doesn't handle any payload... send_object_usage or something like this